scaleway/resource_compute_instance_ip_test.go

```diff
@@ -144,3 +245,28 @@ var testAccScalewayComputeInstanceIPZoneConfig = []string{
 		}
 	`,
 }
+
+func testAccCheckScalewayComputeInstanceServerConfigIP(attachedBase string) string {
```

Comment:
Just matter of personal opinion but I prefer plain config rather than generated one. It's way more complex to understand the test.